microprofile/4.1/metrics/3.0/21.0.0.9-TCKResults.adoc

```diff
@@ -0,0 +1,766 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Metrics 3.0.
+
+== Open Liberty 21.0.0.9 - MicroProfile Metrics 3.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/21.0.0.9/openliberty-runtime-21.0.0.9.zip[Open Liberty 21.0.0.9]
+
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-metrics-3.0/microprofile-metrics-spec-3.0.html[MicroProfile Metrics 3.0]
+
+* Public URL of TCK Results Summary:
++
+link:21.0.0.9-TCKResults.html[TCK results summary]
+
+* Java runtime used to run the implementation:
++
+Java 8 and 11
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+RHEL 8
+
+Java 8 Test results:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8" ?>
+<testsuites>
```

Comment:
This result does not contain the running machine details.

Reply:
Looks like the MicroProfile Metrics TCK set up does not pull in this info. I will talk with the MP Metrics to have this issue addressed.